docs/usage/ipv6.md

```diff
@@ -1,38 +1,22 @@
 # DualStack Support for Gardener GCP Extension
```

Comment:
Not a native speaker, but in the AWS provider documentation we used `dual-stack` a lot instead of `DualStack`. We should probably be consistent.

@n-boshnakov Can you provide guidance in this regard?

Reply:
As the more commonly used way of writing appears to be `dual-stack`, I'd suggest we stay consistent with the AWS documentation. This screenshot provides a good example on using the term in different cases - as a title, in the beginning of a sentence and within a sentence.

![Screenshot 2025-03-21 165359](https://github.com/user-attachments/assets/4e914272-9add-4244-8b5d-ac528fa3e651)